allocate_test.go

```diff
@@ -290,3 +290,28 @@ func TestCombinedOutputError(t *testing.T) {
 		t.Fatalf("got %q, want %q", got, want)
 	}
 }
+
+func TestEnv(t *testing.T) {
+	t.Parallel()
+
+	tz := "Australia/Melbourne"
+	allocCtx, cancel := NewExecAllocator(context.Background(),
+		append([]ExecAllocatorOption{
+			Env("TZ=" + tz),
+		}, allocOpts...)...)
+	defer cancel()
+
+	ctx, cancel := NewContext(allocCtx)
+	defer cancel()
+
+	var ret []byte
```

Comment:
use a `string` here? then it's going to do the JSON unmarshal, so the string will be unquoted

Reply:
Thanks that's a lot simpler.